WinUIGallery/ControlPages/ShapePage.xaml

```diff
@@ -0,0 +1,145 @@
+                        StepFrequency="0.5" SmallChange="1" IsFocusEngagementEnabled="False"/>
+                    <Slider x:Name="slider3" Header="Stroke Thickness" Minimum="2" Maximum="10"
+                        StepFrequency="0.5" SmallChange="1" IsFocusEngagementEnabled="False"/>
+                </StackPanel>
+            </local:ControlExample.Options>
+
+            <local:ControlExample.Xaml>
+                <x:String>
+                    &lt;Ellipse Fill="SteelBlue" Height="$(Slider1)" Width="$(Slider2)" StrokeThickness="$(Slider3)" Stroke="Black"/&gt;
+                </x:String>
+
+            </local:ControlExample.Xaml>
+            <local:ControlExample.Substitutions>
+                <local:ControlExampleSubstitution Key="Slider1" Value="{x:Bind slider1.Value, Mode=OneWay}" />
+                <local:ControlExampleSubstitution Key="Slider2" Value="{x:Bind slider2.Value, Mode=OneWay}" />
+                <local:ControlExampleSubstitution Key="Slider3" Value="{x:Bind slider3.Value, Mode=OneWay}" />
+            </local:ControlExample.Substitutions>
+        </local:ControlExample>
+
+        <!-- RECTANGLE SAMPLE -->
+        <local:ControlExample x:Name="Example2" HeaderText="Rectangle">
+            <Rectangle Fill="SteelBlue"
+           Height="{x:Bind recSlider1.Value, Mode=TwoWay}"
+           Width="{x:Bind recSlider2.Value, Mode=TwoWay}"
+           Stroke="Black"
+           StrokeThickness="{x:Bind recSlider3.Value, Mode=TwoWay}"
+           RadiusY="{x:Bind recSlider4.Value, Mode=TwoWay}"
+           RadiusX="{x:Bind recSlider5.Value, Mode=TwoWay}"/>
+
+            <local:ControlExample.Options>
+                <StackPanel Width="220">
+                    <Slider x:Name="recSlider1" Header="Height" Minimum="100" Maximum="150" 
+                        StepFrequency="0.5" SmallChange="1" IsFocusEngagementEnabled="False"/>
+                    <Slider x:Name="recSlider2" Header="Width" Minimum="100" Maximum="150"
+                        StepFrequency="0.5" SmallChange="1" IsFocusEngagementEnabled="False"/>
+                    <Slider x:Name="recSlider3" Header="Stroke Thickness" Minimum="2" Maximum="10"
+                        StepFrequency="0.5" SmallChange="1" IsFocusEngagementEnabled="False"/>
+                    <Slider x:Name="recSlider4" Header="Radius Y" Minimum="0" Maximum="100"
+                        StepFrequency="0.5" SmallChange="1" IsFocusEngagementEnabled="False"/>
+                    <Slider x:Name="recSlider5" Header="Radius X" Minimum="0" Maximum="100"
+                        StepFrequency="0.5" SmallChange="1" IsFocusEngagementEnabled="False"/>
+                </StackPanel>
+            </local:ControlExample.Options>
+
+            <local:ControlExample.Xaml>
+                <x:String xml:space="preserve">
+&lt;Rectangle Fill="SteelBlue" Height="$(Slider1)" Width="$(Slider2)"
+Stroke="Black" StrokeThickness="$(Slider3)"
+RadiusY="$(Slider4)" RadiusX="$(Slider5)"/&gt;
```

Comment:
nit: formatting

Reply:
@chingucoding is it just a question of adding a tab for the 2nd and 3rd lines or is there something else?